### PR TITLE
[Shop] [Admin] Move liip imagine routing to the main app

### LINKED ISSUE #7984: [Dependencies] [RFC] Liip images routing is defined only for ShopBundle
Routing for Liip images is defined in shop bundle, but we are using it in admin also if I'm correct. Probably should be moved to the main routing file(similar to payum). It will also remove locale prefix in the image path.

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets | fixes #7984 |
| License         | MIT |

### CODE REVIEW

**Comment by lchrusciel** [comment]
The benefit is that you don't have to carry about adding these lines to your setup when you plug in shop or admin. Otherwise, it will probably break. It doesn't matter if both are turned on. But I can agree that putting it in `app/config/routing.yml` is probably the cleanest solution.

**Comment by pamil** [comment]
> It doesn't matter if both are turned on.

If there are the same two routes, the last wins. Currently there is only one route imported by ShopBundle routing, this PR introduces another one in AdminBundle, but changes nothing, since admin routing is imported *before* shop routing. 

Moving it to `app/config/routing.yml` will make sure we won't have any issues with image urls being behind admin firewall if routing loading order changes. Also, since we've introduced localised urls for shop, there are multiple urls leading to the same resource now (eg. `en_US/image.jpg`, 'de_DE/image.jpg`) and it complicates caching them for sure.

**Comment by lchrusciel** [comment]
@pamil updated.

**Comment by pjedrzejewski** [comment]
@lchrusciel This is a BC break, we should mention it in the upgrade file and also upgrade Sylius-Standard.

**Comment by lchrusciel** [comment]
I will resolve it this weekend

File: UPGRADE.md [hunk 1]
```diff
@@ -181,6 +181,13 @@
 
 * `ContactController` has been made final, use decoration instead of extending it.
 
+* `_liip_imagine` routing import has been removed and move to the main routing of Sylius app. If you are not importing `app/config/routing.yml` you need to add this import by your own:
+
+```yml
+_liip_imagine:
+    resource: "@LiipImagineBundle/Resources/config/routing.xml"
+ ```
```
Comment (pamil) [inline UPGRADE.md]: Extra space here

Comment (lchrusciel) [reply UPGRADE.md]: done

**Comment by Zales0123** [comment]
Thank you Łukasz! ⚙️

**Pull request merged**